ext/node/lib.rs

```diff
@@ -350,6 +350,7 @@ deno_core::extension!(deno_node,
     "_next_tick.ts",
     "_process/exiting.ts",
     "_process/process.ts",
+    "_process/report.ts",
```

Comment:
I think the path should be `internal/process/report.ts` to align the file structure as close to Node.js `lib` directory as possible. https://github.com/nodejs/node/tree/main/lib/internal/process

Reply:
I wasn't aware of it, will do!